### PR TITLE
 creating a data model

### PR DESCRIPTION
**Pull request by josuerushanika**
In this MilleStone i did:

- [ ] Create a new app. Give it a meaningful name.
- [ ] Initialize your project with Git.
- [ ] Make sure that project has Postgres database set up.
- [ ] Build project schema.
- [ ]  Create and run the necessary migration files.
- [ ]  Table and column names should match the ERD diagram.
- [ ]  photo for users table should be a link to an image
- [ ]  Foreign keys should be included.
- [ ]  Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- [ ]  All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20230523154812_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create the comments table
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (mikethreels) [inline db/migrate/20230523154812_create_comments.rb:7]: - [ ] you should reference to author as well similar to what you did in the Post table:
```
t.references :author, null: false, foreign_key: { to_table: :users }, index: true
```

File: db/migrate/20230523170858_create_like.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Migration to create the likes table
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```
Comment (mikethreels) [inline db/migrate/20230523170858_create_like.rb:7]: - [ ] you should reference to author as well similar to what you did in the Post table:
```
t.references :author, null: false, foreign_key: { to_table: :users }, index: true
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Josue, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job creating the tables.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @josuerushanika, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to make changes to the tables.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
Thank you @elisha2kyakpo1

**Pull request merged**